apx/src/qapx_stream.cpp

```diff
@@ -632,16 +632,16 @@ void QApxOStreamBuf::apxEndTextMsg()
 
 void QApxOStreamBuf::apxNodeNameRqst()
 {
-   mBuf.append("?N*\n");
```

Comment:
A lot of this file is old left overs from when I experimented with the APX 1.1 query language. None of it is in use today and will probably be removed later.